### ISSUE TITLE
[MSFT-248] Sample challenge page: Challenge prerequisites popup: Unlabelled button at the end of the popup.

### ISSUE THREAD

**Issue by veshu** [importtest/ewe#164]
**Steps to Reproduce**
1. Go to the Sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure that the screen reader is switched on. 3. activate the register button so that the challenge prerequisites popup is displayed. 4. Press the key combination 'ctrl + insert + b' to bring up a list of buttons on the page. 5. Note there is an unlabelled button at the end of this list. 6. This button when activated closes the challenge prerequisites popup.

**Expected Result**
Screen reader should announce the appropriate label for the button.

**Actual Result**
Screen reader reads an unlabelled button at the end of the challenge prerequisites popup which when activated closes the popup.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2042
